app/models/application_form.rb

```diff
@@ -162,15 +162,7 @@ class ApplicationForm < ApplicationRecord
 
   STATUS_COLUMNS.each { |column| enum column, STATUS_VALUES, prefix: column }
 
-  scope :assessable, -> { where.not(stage: %i[draft completed]) }
```

Comment:
I think this scope is used in a number of other places so we'll need to keep it:

https://github.com/search?q=repo%3ADFE-Digital%2Fapply-for-qualified-teacher-status%20assessable&type=code

Reply:
Oh yes...that's fixed now